### PR TITLE
Docs/SK-901 | MonAI README

### PR DESCRIPTION
**Pull request by sztoor**
## Description
This branch mainly introduces a new README file to integrate this example into the MonAI tutorial repository. To enhance user understanding, we simplified the get and load data functions. This involved code modifications and more extensive updates to the README document. Now, I have fixed the code checks as well.

### CODE REVIEW

**Review by ahellander — APPROVED** [review]

File: examples/monai-2D-mednist/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+# Implementing 2D Classification Model with MedNIST Dataset Using FEDn  
+
+This example provides a step-by-step guide to deploying and running a 2D classification model using the MedNIST dataset in a federated environment with the [FEDn framework|](https://www.scaleoutsystems.com/framework) developed by [Scaleout Systems](https://www.scaleoutsystems.com/). This example builds on the centralized [example](https://github.com/Project-MONAI/tutorials/blob/main/2d_classification/mednist_tutorial.ipynb) from the MonAI project and adapts it for federated learning settings, utilizing the same code for ease of transition. 
+
+The FEDn framework supports researchers with its robust [open-source SDK](https://fedn.readthedocs.io/en/stable/quickstart.html) and a [public SaaS platform](https://fedn.readthedocs.io/en/stable/studio.html), enabling scalable and efficient federated learning use cases. 
+
+# Getting Started
+
+
+For a step-by-step example guide, click [here](https://github.com/scaleoutsystems/fedn/tree/master/examples/monai-2D-mednist>).
```
Comment (Wrede) [inline examples/monai-2D-mednist/README.md:11]: this link will get you to the same README (this one)

File: examples/monai-2D-mednist/README.rst
Comment (Wrede) [inline examples/monai-2D-mednist/README.rst:1]: Why are there 1 .md and 2 .rst files?

**Review by Wrede — CHANGES_REQUESTED** [review]

File: examples/monai-2D-mednist/README.rst
Comment (sztoor) [reply examples/monai-2D-mednist/README.rst:1]: I will remove .rst from the branch as in MonAI tutorials repo, they have all .md files.

File: examples/monai-2D-mednist/client/train.py [hunk 2]
```diff
@@ -54,18 +53,16 @@ def train(in_model_path, out_model_path, data_path=None, client_settings_path=No
...
             client_settings = dict(yaml.safe_load(fh))
         except yaml.YAMLError:
             raise
 
-    print("client settings: ", client_settings)
     batch_size = client_settings["batch_size"]
     max_epochs = client_settings["local_epochs"]
     num_workers = client_settings["num_workers"]
-    split_index = client_settings["split_index"]
+    split_index = os.environ.get("FEDN_DATA_SPLIT_INDEX")  #client_settings["split_index"]
```
[the remark this answers is not in the document]
Comment (sztoor) [reply examples/monai-2D-mednist/client/train.py:65]: Done!

File: examples/monai-2D-mednist/client/validate.py [hunk 3]
```diff
@@ -45,7 +45,7 @@ def validate(in_model_path, out_json_path, data_path=None, client_settings_path=
 
     num_workers = client_settings["num_workers"]
     batch_size = client_settings["batch_size"]
-    split_index = client_settings["split_index"]
+    split_index = os.environ.get("FEDN_DATA_SPLIT_INDEX")  # client_settings["split_index"]
```
[the remark this answers is not in the document]
Comment (sztoor) [reply examples/monai-2D-mednist/client/validate.py:48]: done.

File: examples/monai-2D-mednist/README.md (hunk 1 above)
Comment (sztoor) [reply examples/monai-2D-mednist/README.md]: In FEDn MonAI example directory, we have "README.rst," which stays the same. The MonAI tutorials repository will contain "README_MonAI_Tutorial.md" in the tutorials/federated_learning/FEDn/ directory as "README.md." This file (README.md) will point to README.rst in the MonAI example directory.

**Pull request closed without merging**